docs/Built-ins.md

```diff
@@ -947,6 +947,24 @@ Defaults to `True`
 
 Defaults to `3600`
 
+**LOCAL MODULES**
```

Comment:
see the note at the top of this file - it's generated so your edits will be lost. You should instead change the stardoc blocks in the source file

Reply:
@alexeagle You're right. Any idea about the best place to put those notes as they are not doc for any attribute?

Reply:
I've updated the docs correctly on the source